includes/rest-api/class-sensei-rest-api-home-controller.php

```diff
@@ -0,0 +1,284 @@
+							'extra_action' => [
+								'title' => 'Upgrade to Sensei Pro',
+								'url'   => 'https://...',
+							],
+							'icon'         => 'lock',
+						],
+					],
+				],
+			],
+			'guides'                      => [
+				// TODO: Load from https://senseilms.com/wp-json/senseilms-home/1.0/{sensei-lms|sensei-pro|interactive-blocks}.json.
+				'items'    => [
+					[
+						'title' => 'How to Sell Online Courses',
+						'url'   => 'http://...',
+					],
+					[
+						'title' => 'How to Creating Video Courses',
+						'url'   => 'http://...',
+					],
+					[
+						'title' => 'How to Choose the Right Hosting Provider',
+						'url'   => 'http://...',
+					],
+				],
+				'more_url' => 'http://senseilms.com/category/guides/',
+			],
+			'news'                        => [
+				// TODO: Load from https://senseilms.com/wp-json/senseilms-home/1.0/{sensei-lms|sensei-pro|interactive-blocks}.json.
+				'items'    => [
+					[
+						'title' => 'Introducing Interactive Videos For WordPress',
+						'date'  => '2022-08-31', // Localized for user.
+						'url'   => 'http://senseilms.com/inroducing-interactive-videos/',
+					],
+					[
+						'title' => 'New Block Visibility, Scheduled Content, and Group Features',
+						'date'  => '2022-08-09', // Localized for user.
+						'url'   => 'http://senseilms.com/conditional-content/',
+					],
+				],
+				'more_url' => 'https://senseilms.com/blog/',
+			],
+			'extensions'                  => [
+				// TODO: Load from https://senseilms.com/wp-json/senseilms-home/1.0/{sensei-lms|sensei-pro}.json.
+				[
+					'title'        => 'Sensei LMS Post to Course Creator',
+					'image'        => 'http://senseilms.com/wp-content/uploads/2022/02/sensei-post-to-course-80x80.png',
+					'description'  => 'Turn your blog posts into online courses.',
+					'price'        => 0,
+					'product_slug' => 'sensei-post-to-course', // To be used with the installation function `Sensei_Setup_Wizard::install_extensions`.
+					'more_url'     => 'http://senseilms.com/product/sensei-lms-post-to-course-creator/',
+				],
+			],
+			'show_sensei_pro_promo'       => false, // Whether we should show the promotional banner for Sensei Pro or not.
+			'notifications'               => [
+				[
+					'title'            => 'Your Sensei Pro license expires on 12.09.2022.',
+					'main_action'      => [
+						'title' => 'Update now',
+						'url'   => 'https://...',
+					],
+					'secondary_action' => [
+						'title' => 'What\'s new',
+						'url'   => 'https://...',
+					],
+					'level'            => 'error', // One of: info, warning, error.
+					'dismissible'      => false, // The default value is true.
+				],
+				[
+					'title'            => 'Good news, reminder to update to latest version',
+					'main_action'      => [
+						'title' => 'Update now',
+						'url'   => 'https://...',
+					],
+					'secondary_action' => [
+						'title' => 'Link for more information',
+						'url'   => 'https://...',
+					],
```

Comment:
I thought about that, but given that the variant will mark also the position I think it would be easier from the frontend to model them separately. I doubt we would get more than main and an optional secondary one.